.github/workflows/cleanup-all.yml

```diff
@@ -1,76 +1,60 @@
 name: Clean Atlas organization
-# local run sample:
-# echo '{"ref":"HEAD/ref/CLOUDP-smth", "ref_type":"branch", "action":"workflow_dispatch", "inputs": {"clean":"true", "timelife": "3"} }' > payload.json | act -j cleanup-all -e payload.json
 
 on:
   schedule:
-    - cron: "0 1 * * 1-5"
-    - cron: "0 3 * * 1-5"
+    - cron: "*/30 7-22 * * 1-5"
   workflow_dispatch:
     inputs:
-      clean:
-        description: "Clean all(true) or old empty projects(false)?"
-        default: "false"
-        required: true
-      timelife:
-        description: "Timelife of project in hours"
+      lifetime:
+        description: "Lifetime of project in hours"
         default: "3"
         required: true
-      cleanPE:
-        description: "Run PE cleaner(true) or not(false)?"
-        default: "true"
-        required: true
-      cleanVPC:
-        description: "Clean VPC(true) or not(false)?"
-        default: "true"
-        required: true
 
 concurrency:
   group: cleanup
 
 jobs:
-  cleanup-all:
-    name: Cleanup atlas empty projects, and terminate cluster
+  cleanup:
+    name: Cleanup Atlas Cloud
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        target: ["CloudQA", "CloudGovQA"]
     steps:
-      - uses: actions/checkout@v4
+      - name: Checkout codebase
+        uses: actions/checkout@v4
 
       - name: Set up Go
         uses: actions/setup-go@v4
         with:
-          go-version-file: "${{ github.workspace }}/go.mod"
+          go-version-file: "${{ github.workspace }}/tools/clean/go.mod"
 
-      - name: Build & Test clean tool
-        run: (cd tools/clean && go build . && go test .)
```

Comment:
Why do we skip self-testing?
This testing will not be run automatically anywhere else AFAIK, which renders the test code pointless.

Reply:
I saw later that tests were removed. Then this makes sense.

On the other hand, this cleanup code is not trivial and lacks any self testing. I guess as it is not production related, we can relax and react if that becomes a problem at some point.